sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-energy-demand/automl-forecasting-task-energy-demand-advanced-mlflow.ipynb

```diff
@@ -533,14 +614,501 @@
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Render the JSON as a pandas DataFrame\n",
+    "with open(os.path.join(local_path, \"featurization_summary.json\"), \"r\") as f:\n",
+    "    records = json.load(f)\n",
+    "fs = pd.DataFrame.from_records(records)\n",
+    "\n",
+    "# View a summary of the featurization\n",
+    "fs[\n",
+    "    [\n",
+    "        \"RawFeatureName\",\n",
+    "        \"TypeDetected\",\n",
+    "        \"Dropped\",\n",
+    "        \"EngineeredFeatureCount\",\n",
+    "        \"Transformations\",\n",
+    "    ]\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 6.2 Forecasting using batch endpoint<a id=\"forecast\"></a>\n",
+    "\n",
+    "Now that we have retrieved the best pipeline/model, it can be used to make predictions on test data. We will do batch inferencing on the test dataset which must have the same schema as training dataset.\n",
+    "\n",
+    "The inference will run on a remote compute. In this example, it will re-use the training compute.\n",
+    "First we will load model and environment from the local file."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a model endpoint\n",
+    "First we need to register the model, environment and the batch endpoint."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import (\n",
+    "    Environment,\n",
+    "    BatchEndpoint,\n",
+    "    BatchDeployment,\n",
+    "    BatchRetrySettings,\n",
+    "    Model,\n",
+    ")\n",
+    "from azure.ai.ml.constants import BatchDeploymentOutputAction\n",
+    "\n",
+    "model_name = \"energy_demand\"\n",
+    "batch_endpoint_name = \"energy-demand\"\n",
+    "\n",
+    "model = Model(\n",
+    "    path=f\"azureml://jobs/{best_run.info.run_id}/outputs/artifacts/outputs/model.pkl\",\n",
+    "    name=model_name,\n",
+    "    description=\"Energy demand model.\",\n",
+    ")\n",
+    "registered_model = ml_client.models.create_or_update(model)\n",
+    "\n",
+    "env = Environment(\n",
+    "    name=\"automl-tabular-env\",\n",
+    "    description=\"environment for automl inference\",\n",
+    "    image=\"mcr.microsoft.com/azureml/openmpi3.1.2-ubuntu18.04:20210727.v1\",\n",
+    "    conda_file=\"artifact_downloads/outputs/conda_env_v_1_0_0.yml\",\n",
+    ")\n",
+    "\n",
+    "endpoint = BatchEndpoint(\n",
+    "    name=batch_endpoint_name,\n",
+    "    description=\"this is a sample batch endpoint\",\n",
+    ")\n",
+    "ml_client.begin_create_or_update(endpoint)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Create a batch deployment we will use the forecasting_script.py which will load the model and will call forecast each time we will envoke the endpoint."
```

Comment:
nit: "To create a batch deployment, ..."

Reply:
Fixed